pom.xml

```diff
@@ -36,25 +38,25 @@
   </licenses>
 
   <scm>
-    <url>https://github.com/jenkinsci/aws-codepipeline-plugin</url>
-    <connection>scm:git:ssh://github.com/jenkinsci/aws-codepipeline-plugin.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/aws-codepipeline-plugin.git</developerConnection>
-    <tag>aws-codepipeline-0.47</tag>
+    <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+    <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+    <url>https://github.com/${gitHubRepo}</url>
+    <tag>${scmTag}</tag>
   </scm>
 
   <repositories>
     <repository>
-        <id>repo.jenkins-ci.org</id>
-        <url>https://repo.jenkins-ci.org/public/</url>
-    </repository>
-    <repository>
-        <id>lib</id>
-        <url>file://${project.basedir}/lib</url>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/public/</url>
     </repository>
   </repositories>
 
   <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <revision>0.48</revision>
```

Comment:
0.47 was a failed release, so choosing the next available version. See https://www.jenkins.io/doc/developer/publishing/releasing-manually/